style.css

```diff
@@ -82,3 +126,207 @@ body {
   font-size: 28px;
   color: var(--neutral-icons);
 }
+
+.works {
+  width: 380px;
+  background-color: #f7f7f9;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(4, 1fr);
+  gap: 70px;
+  padding: 20px;
+}
+
+.works .card {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-around;
+  margin: 0 auto;
+  width: 327px;
+  height: 552px;
+  background-color: #fff;
+  padding: 8px 16px;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.works .card img {
+  width: 295px;
+  height: 200px;
+  border-radius: 8px;
+  object-fit: cover;
+}
+
+.card h1 {
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 700;
+  color: var(--neutral-card);
+  padding: 8px 0;
+}
+
+.card .card-details {
+  display: flex;
+  align-items: center;
+}
+
+.card .card-details h3 {
+  font-size: 13px;
+  line-height: 16px;
+  font-weight: 600;
+  color: var(--neutral);
+  text-transform: uppercase;
+}
+
+.card .card-details i {
+  font-size: 8px;
+  color: var(--neutralsN50);
+  padding: 0 13px;
+}
+
+.card .card-details h4 {
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: capitalize;
+  font-weight: 600;
+  color: var(--neutralsN100);
+}
+
+.card .card-description {
+  padding-top: 10px;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  color: var(--neutral);
+}
+
+.works .card-skills {
+  padding: 20px 0;
+}
+
+.works .card-skills ul {
+  width: 200px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.works .card-skills ul li {
+  padding: 1px 10px;
+  background-color: var(--primaryB50);
+  margin: 0 3px;
+  border-radius: 8px;
+}
+
+.works .card-skills ul li a {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: var(--primary);
+}
+
+.about {
+  width: 380px;
+  background-color: #fff;
+  padding: 80px 20px;
+}
+
+.about h1 {
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700;
+  color: var(--neutral-card);
+}
+
+.about p {
+  padding: 10px 0;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 400;
+  color: var(--neutral);
+}
+
+.connect {
+  text-transform: uppercase;
+  color: var(--secondary);
+  font-weight: 500;
+  font-size: 16px;
+}
+
+.about .main-icons ul {
+  display: flex;
+  justify-content: space-between;
+  padding: 14px 0;
+  width: 180px;
+}
+
+.about .main-icons ul li a {
+  font-size: 22px;
+  color: var(--neutral-icons);
+}
+
+.about button {
+  margin-bottom: 35px;
+  text-transform: none;
+}
+
+.about .main-skill {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 90%;
+  cursor: pointer;
+}
+
+.about .main-skill i {
+  font-size: 20px;
+  margin-right: 30px;
+}
+
+.about .main-skill h2 {
+  font-size: 18px;
+  line-height: 24px;
+  color: #000;
+  font-weight: 500;
+}
```

Comment:
- [ ] Good job implementing these styles from Figma. However, the `font-size` on Figma design is `20px`. consider using the correct `font-size` so as to match your chosen design template.